### PR TITLE
Production Consumption Toggle / mobile modal fix

### PR DESCRIPTION
**Pull request by ovbm**
the toggle on mobile:

notice that I removed the top electricitymap logo to gain some vertical space and instead placed the logo in the tab of the app. I think this solution makes a lot of sense for mobile:
emapanim

What's missing is a modal or legend that explains what's happening, maybe with a link to a blogpost for further info. Can someone make a suggestion for this text?

todo:
- [x] explanation moda/legend

### CODE REVIEW

**Comment by corradio** [comment]
Smart move. Small detail: I see an vertical alignment issue with the caption of the button compared to the two other buttons.
Do you think we should rename "Areas"?
For the modal, we don't have any blogpost, but for consumption we could say "takes into account imports and exports" and for production "ignores imports and exports"

File: web/public/images/electricitymap-icon.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (corradio) [inline web/public/images/electricitymap-icon.svg]: didn't we already have an icon?

File: web/src/main.js [hunk 2]
```diff
@@ -794,6 +794,15 @@ if (!getState().application.isMobile) {
   });
 }
 
+// Production/Consumption
```
Comment (corradio) [inline web/src/main.js]: can you also remove the toggle that I coded in the bottom of main.js (that binds to `window`?)

File: web/src/main.js [hunk 3]
```diff
@@ -794,6 +794,15 @@ if (!getState().application.isMobile) {
...
+function toggleElectricityMixMode() {
+  dispatchApplication('electricityMixMode', getState().application.electricityMixMode === 'consumption'
+    ? 'production'
+    : 'consumption');
+  const activeToggle = d3.select('.production-toggle-active-overlay');
```
Comment (corradio) [inline web/src/main.js]: This should be in the observer instead of here.

File: web/public/images/electricitymap-icon.svg (hunk 1 above)
Comment (ovbm) [reply web/public/images/electricitymap-icon.svg]: no, this is just the emap symbol as svg, we only had emap smybol + text as svg

**Comment by ovbm** [comment]
The vertical alignment was always there, it moves the "active" legend item a couple of pixels up compared to the inactive ones. not a fan but I kept it because it was already there.

Don't know about Areas, do you have an Idea?

Ok, so it's going to be a hover label with those two infos. I wonder if the toggle is too prominent and if it should be collapsed.

File: web/src/main.js (hunk 3 above)
Comment (ovbm) [reply web/src/main.js]: do you mean the activeToggle part?

**Comment by ovbm** [comment]
this is a solution for the mobile itunes thing:
screenshot 2018-10-30 at 10 52 49
screenshot 2018-10-30 at 10 52 27

File: web/public/images/electricitymap-icon.svg (hunk 1 above)
Comment (corradio) [reply web/public/images/electricitymap-icon.svg]: ok!

File: web/src/main.js (hunk 3 above)
Comment (corradio) [reply web/src/main.js]: yes

**Comment by corradio** [comment]
yes indeed default should be consumption. OK for the other changes.

**Comment by corradio** [comment]
you should take a look at the reducer where default values are set

**Comment by ovbm** [comment]
I think my work is done for this now

File: web/src/main.js (hunk 2 above)
Comment (ovbm) [reply web/src/main.js]: done

File: web/src/main.js (hunk 3 above)
Comment (ovbm) [reply web/src/main.js]: done

**Comment by corradio** [comment]
Merging now, we can always improve later.

**Pull request merged**